frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopover.tsx

```diff
@@ -238,6 +242,8 @@ export default function FilterPopover({
   const shouldShowDatePicker = field?.isDate() && !field?.isTime();
   const supportsExpressions = query.database()?.supportsExpressions();
 
+  const hasEmptyPicker = filter.operator()?.fields.length === 0;
```

Comment:
How about

```suggestion
  const isPickerEmpty = filter.operator()?.fields.length === 0;
```

Reply:
Actually, this does not affect children of `FilterPopoverPicker`, but rendering of `FilterPopoverPicker` itself.
So maybe this would be better?

```suggestion
  const hasPicker = filter.operator() & filter.operator().fields.length > 0;
```